### DISCUSSION TITLE
Largest Contentful Paint score changes in HTTPS

### DISCUSSION THREAD

**Discussion by debjits1** [GoogleChrome/lighthouse#12409] [Q&A]
I am loading the Skeleton of a carousel with server-side rendering and loading the actual carousel with images at client-side render. Finally, the server-rendered skeleton is removed from the DOM. 
The above implementation is showing LCP result of 0.8s. But, when I load my page with HTTPS, it shows the LCP time as 11s. Does anyone have any idea what can be the reason?

**Comment by connorjclark** [comment]
resources being blocked due to mixed content?

**Reply by debjits1** [reply]
@connorjclark how do I check if any resource is being blocked?

**Reply by connorjclark** [reply]
DevTools network panel.

Please share a URL.

**Reply by debjits1** [reply]
@connorjclark  I have checked the network panel. No request is being blocked. Is there any other possible reason for this? 

I am actually working in a private repo, so I am not able to share the URL
